pkg/wasm/convert.go

```diff
@@ -38,23 +41,104 @@ var (
 	allowHTTPTypedConfig = &anypb.Any{
 		TypeUrl: "type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC",
 	}
-	allowNetworkTypedConfig = &anypb.Any{
-		TypeUrl: "type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC",
-	}
+	denyHTTPTypedConfig, _ = anypb.New(&httprbac.RBAC{
+		Rules: &rbacv3.RBAC{
+			Action: rbacv3.RBAC_DENY,
+			Policies: map[string]*rbacv3.Policy{
```

Comment:
Do we need to specify these policies?

Reply:
Do you have better options?
This is also a point I was uncertain about when implementing this here. Currently, this implementation leads to the return of 'access denied by rbac', and using fault injection is also a good option.

Reply:
Do you have better options?
This is also a point I was uncertain about when implementing this here. Currently, this implementation leads to the return of 'access denied by rbac', and using fault injection is also a good option.

Reply:
cannot you leave it empty

Reply:
I was thiniking leaving it empty with DENY Action should help as well